package.json

```diff
@@ -257,26 +258,26 @@
   },
   "devDependencies": {
     "@types/glob": "^7.1.1",
-    "@types/mocha": "^5.2.6",
```

Comment:
@mohitsuman ideally upgrades to dependencies should be in a separate commit so that - if required - we can undo things separately.